react/src/lib/List/index.js

```diff
@@ -174,29 +174,45 @@ class List extends React.Component {
         flag = true;
         break;
 
-      case 38:
-      case 37:
-        this.getNextFocusedChild(items, tgt, -1);
-        this._needsRefocus = true;
-        if(!shouldPropagateKeyDown) flag = true;
+      case 38: // UP
+        if (navigationDirection === 'both' || navigationDirection === 'vertical') {
```

Comment:
Might consider assigning `navigationDirection === 'both' || navigationDirection === 'vertical'` to a variable to make clear the intent. `isVerticalOrAny`? Same goes for when you're checking for `horizontal`.

Reply:
Fair will make appropriate changes.